include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -495,6 +495,54 @@ struct edit_distance_unbanded_trace_matrix_policy :
     //!\}
 };
 
+//!\brief The same as `T &` but it is default constructible and is re-assignable.
+template <typename T>
```

Comment:
Template Parameter not documented. Doxygen ingroup?

Reply:
Not sure which group it should belong to